app/filtering.js

```diff
@@ -367,7 +374,7 @@ function registerPermissionHandler (session, partition) {
       cb(result)
       if (persist) {
         // remember site setting for this host over http(s)
-        appActions.changeSiteSetting(getOrigin(url), permission + 'Permission', result, isPrivate)
+        appActions.changeSiteSetting(origin, permission + 'Permission', result, isPrivate)
```

Comment:
need to strip the trailing `/` from origin for backwards compatibility; i think this is why tests are breaking